cypress.config.ts

```diff
@@ -1,5 +1,6 @@
 import registerCodeCoverageTasks from "@cypress/code-coverage/task";
 import { defineConfig } from "cypress";
+import readPdf from "./cypress/support/readPdf";
```

Comment:
can we get this as an @ import

Reply:
If I use an @ import it will look like "@/../../cypress/support/readPdf" and cypress doesn't seem to like it when I put it in the components folder and put an @ either. Discussed with @SamYap-Softwire and I think for configs we dont use @ import. If anyone has more insights then please lmk!